content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Notary Project announces its first major release!
+author:  "Notary Project Release Team"
+date: 2023-08-22
+draft: false
+---
+
+The Notary Project maintainers are proud to announce its first major release, including [Notary Project specifications v1.0.0](https://github.com/notaryproject/specifications/releases/tag/v1.0.0), [notation v1.0.0](https://github.com/notaryproject/notation/releases/tag/v1.0.0), [notation-go v1.0.0](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0), and [notation-core-go v1.0.0](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0) which are ready for production use!
+
+## What is Notary Project and Notation?
+
+As containers and cloud native artifacts become common deployment units, users want to make sure that cloud native artifacts in their environments are authentic and not tampered with. The Notary Project is a set of specifications and tools intended to provide cross-industry standards for securing software supply chains through signing and verification, signature portability, and key/certificate management. 
```

Comment:
> intended to provide cross-industry standards

Does the Notary project actually "provide" the standards? I would think it would be more accurate to say:

"intended to implement cross-industry standards"